GoogleForCreators/web-stories-wp
# Supporting qualifying outbound links (rel attribute)

## Context



## Summary

For certain links users might want to tell search engines their relationship with the linked page using the rel attribute. Examples include:

rel="sponsored" - Mark links that are advertisements or paid placements
rel="ugc" - Mark links as user-generated content (UGC) such as comments and forum posts
rel="nofoloow" - For when you'd rather Google not associate your site with, or crawl the linked page from, your site

## Relevant Technical Choices



## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/141502140-33134a6f-a6b8-4516-bca2-628c8a7ed8e4.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Create a story.
2. Insert image into story. 
3. In design panel, add link, example. http://www.google.com
4. Checkbox no follow or other option. 
5. Publish story. 
6. View story on front end. 
7. View source. 
8. Find `<a` tag and see rel value. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #4440
